tasks/gen.go

```diff
@@ -138,6 +140,7 @@ func main() {
 	ts.Accumulate(schema.SpawnStruct("PopTask", []schema.StructField{
 		schema.SpawnStructField("Status", "Status", false, false),
 		schema.SpawnStructField("WorkedBy", "String", false, false),
+		schema.SpawnStructField("Tags", "String", true, false),
```

Comment:
I think this should be called AnyTags or AllTags, to clarify if this is "at least one tag is shared with the task to pop" or "all tags must be shared with the task to pop".

Reply:
Oh, and assuming this is a list too, I'd also make it an actual IPLD list.

Reply:
Done